db/migrate/20220124191056_add_type_to_cdn_configuration.rb

```diff
@@ -1,15 +1,20 @@
 class AddTypeToCdnConfiguration < ActiveRecord::Migration[6.0]
+  class FakeCdnConfiguration < Katello::Model
+    self.table_name = 'katello_cdn_configurations'
+    self.inheritance_column = nil
+  end
+
   def change
     add_index :katello_cdn_configurations, :organization_id, unique: true
-    add_column :katello_cdn_configurations, :type, :string, default: ::Katello::CdnConfiguration::CDN_TYPE
+    add_column :katello_cdn_configurations, :type, :string, default: 'redhat_cdn'
 
-    ::Katello::CdnConfiguration.reset_column_information
-    ::Katello::CdnConfiguration.all.each do |config|
+    FakeCdnConfiguration.reset_column_information
+    FakeCdnConfiguration.all.each do |config|
       unless config.username.blank? ||
              config.password.blank? ||
              config.upstream_organization_label.blank? ||
              config.ssl_ca_credential_id.blank?
-        config.update!(type: ::Katello::CdnConfiguration::UPSTREAM_SERVER_TYPE)
+        config.update!(type: 'upstream_server')
```

Comment:
`network_sync`?

Reply:
Good point. Not really. But upstream users may have, so better to be safe imo.

Reply:
Actually ```::Katello::CdnConfiguration::UPSTREAM_SERVER_TYPE)``` no longer exists after this PR. We should not be failing any one running this migration, hence changed it to its string form.